### DISCUSSION TITLE
Ikea LED2005R5/LED2004G8 issue

### DISCUSSION THREAD

**Discussion by oleksii-kalinin** [Koenkk/zigbee2mqtt#16441] [Devices]
Hi. I have zibee2mqtt installed with a conbee2 adapter with some Ikea lamps LED2005R5/LED2004G8. That lamps are grouped with z2m. The issue is that lights do not always respond to the mqtt state. That means that they can stay turned off or not turn off. In logs, I clearly see that z2m updated the state to 'On' or 'Off', but the lamps are still in the previous state. Lamps can be turned on or off after a few minutes.

**Comment by wgentine** [comment]
Having exactly same issue. Moved from LED1837R5 to LED2005R5 for the color_temp and started having all these.

**Reply by oleksii-kalinin** [reply]
My issues were fixed by updating `flux` integrations to bigger intervals between color temp changes.

**Comment by ChristianPorsche** [comment]
I'm having the same issue. I see you fixed the issue by updating the flux. Where can I do that?
